client/rest-high-level/src/main/java/org/elasticsearch/client/SecurityClient.java

```diff
@@ -885,6 +887,36 @@ public void createApiKeyAsync(final CreateApiKeyRequest request, final RequestOp
                 CreateApiKeyResponse::fromXContent, listener, emptySet());
     }
 
+    /**
+     * Retrieve API Key(s) information.<br>
+     * See <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-get-api-key.html">
+     * the docs</a> for more.
+     *
+     * @param request the request to retrieve api key(s)
```

Comment:
nit: Not sure what we should use from `Api Key`, `API Key`, `api key`, `API KEY` but let's try to be consistent